source/layers/tracing/ze_tracing_cb_structs.h

```diff
@@ -30,6 +30,11 @@ typedef struct _zel_global_callbacks_t
 /// @brief Table of RTASBuilder callback functions pointers
 typedef struct _zel_rtas_builder_callbacks_t
 {
+    ze_pfnRTASBuilderCreateExtCb_t                                  pfnCreateExtCb;
```

Comment:
@nrspruit @vishnu-khanth  I feel this is odd that we're adding members to the start of a structure, shouldn't they be at the end.. Is this weird artifact of this temp branch or did we mess something up with the RTASBuilder extension promotion...... Do we even care with this specific structure if totally internal.

Reply:
because these are assigned individually, the ordering of the callbacks is not important.